src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -893,14 +893,18 @@ private async Task<HttpResponseMessage> HandleStartOrchestratorRequestAsync(
                         OrchestrationInstance = instance,
                     };
 
-                    string traceParent = GetHeaderValueFromHeaders("traceparent", request.Headers);
-                    string traceState = GetHeaderValueFromHeaders("tracestate", request.Headers);
+                    string traceParent = GetHeaderValueFromHeaders("x-client-traceparent", request.Headers);
+                    string traceState = GetHeaderValueFromHeaders("x-client-tracestate", request.Headers);

```

Comment:
What is this change for? I thought the previous header names were the official ones per the W3C spec?